discovery-provider/src/tasks/entity_manager/entities/social_features.py

```diff
@@ -78,9 +78,15 @@ def create_social_record(params: ManageEntityParameters):
                 is_delete=False,
             )
         elif record_type == EntityType.SAVE:
-            create_record = create_save(params)
+            if not get_attribute_from_record_metadata(
```

Comment:
i was expecting this to just check the existing record in the validation

https://github.com/AudiusProject/audius-protocol/blob/b3cd997dbc1a1edbdb1ab9720f1a6ff0af1fe257/discovery-provider/src/tasks/entity_manager/social_features.py#L256

does that make sense? then the metadata isn't necessary.

Reply:
especially when you consider the case when someone uses SDK and just says i'm reposting the track and pretends it's hidden. indexing should validate that against the actual existing track not the metadata that's passed in

Reply:
yea that def makes sense. i didn't realize i had access to the entity itself which is why i went the first route but i like this new route much better